.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "tabWidth": 2
```

Comment:
This config file may be removed, but keeping it here will allow consistent code style.